src/main.ts

```diff
@@ -21,12 +23,19 @@ export enum DependabotErrorType {
 export async function run(context: Context): Promise<void> {
   try {
     core.info('🤖 ~ starting update ~')
+
+    if (context.actor !== DEPENDABOT_ACTOR) {
+      core.info('This workflow can only be triggered by Dependabot.')
+      core.info('🤖 ~ finished: nothing to do ~')
+      return // TODO: This should be setNeutral in future
```

Comment:
Per [the documentation](https://github.com/actions/toolkit/tree/main/packages/core#exit-codes) setNetural isn't supported yet.

Showing success is misleading, but failing the job has implications for Dependabot's error handling so this is preferable for now.